cli.js

```diff
@@ -19,6 +19,7 @@ const cli = meow(`
 	  --any-branch  Allow publishing from any branch
 	  --no-cleanup  Skips cleanup of node_modules
 	  --yolo        Skips cleanup and testing
+	  --no-checks   Skips verifying status checks on GitHub
```

Comment:
`--no-checks` is too generic. How about `--no-gh-checks`?

Reply:
Since `np` recently added support for GitLab, I wanted to keep the option open to check [GitLab’s commit statuses](https://docs.gitlab.com/ee/api/commits.html#list-the-statuses-of-a-commit) in the future as well. What do you think about having both of these checks under `--no-checks` (or perhaps `--no-status-checks` to be a bit more specific)?

Reply:
Renamed in fc65b0fb8fa4b8bcdc2f61fd086b6fdb1269c35c 🙂